modules/benchmark/R/metric.residual.plot.R

```diff
@@ -8,21 +8,24 @@
 metric.residual.plot <- function(dat, var){
   
   require(ggplot2)
+  # 
+  # ind <- intersect(which(!is.na(dat$obvs)),  which(!is.na(dat$model)))
+  # 
+  # dat <- dat[ind,]
   
-  ind <- intersect(which(!is.na(dat$obvs)),  which(!is.na(dat$model)))
-  
-  dat <- dat[ind,]
-  dat$time <- lubridate::year(as.Date(as.character(dat$time), format="%Y",))
-  dat$diff <- dat$model - dat$obvs
+  dat$time <- lubridate::year(as.Date(as.character(dat$time), format="%Y"))
+  dat$diff <- abs(dat$model - dat$obvs)
+  dat$zeros <- rep(0, length(time))
   
-  ggplot(data = dat) + 
-    geom_path(aes(x=time,y=rep(0, length(time))), colour = "#666666", size=2, linetype = 2, lineend = "round") +
-    geom_point(aes(x=time,y=diff), size=4,  colour = "#619CFF") + labs(title=var, x= "years", y="model - observation")
+  p <- ggplot(data = dat, aes(x=time)) + 
+    geom_path(aes(y=zeros), colour = "#666666", size=2, linetype = 2, lineend = "round") +
+    geom_point(aes(y=diff), size=4,  colour = "#619CFF") + labs(title=var, x= "years", y="abs(model - observation)")
   
+  plot(p)
```

Comment:
I've never seen this before. If it works, great, but usually one plots ggplots using `print`, not `plot`.